README.md

```diff
@@ -49,8 +49,8 @@ You first need to create a Python class that inherits from the SDK's `panoply.Da
 import panoply
 
 class MyDataSource(panoply.DataSource):
-  def __init__(self, source, options):
-    super(MyDataSource, self).__init__(source, options)
+  def __init__(self, *args, **kwargs):
+    super(MyDataSource, self).__init__(*args, **kwargs)
```

Comment:
You can use whatever you want, but it is advised to append `*args, **kwargs` at the end and pass them to the super constructor.